### ISSUE TITLE
Add better hadling for selection in interactive menu (Skip,Abort, etc.) so they won't be marked as "completed"

### ISSUE THREAD

**Issue by knuurr** [sentriz/betanin#120]
Currently, imports where manual action was needed, and during which Beets was told to either Skip that particual album, or Abort import, are detected by UI as success:

![obraz](https://github.com/sentriz/betanin/assets/135069967/bf8af4d0-2c7b-4ac7-b0b7-6aae3114ecb1)

![obraz](https://github.com/sentriz/betanin/assets/135069967/58ea3566-e788-4aa2-b0ae-09a5fef59ca8)

Now I get it that it's probably based on exit code, and probably Beets doesn't dare to report Aborted import as non-zero.

IDEA: would it be possible to check at betanin level, that:
- check if Beets printed prompt about action to be taken, 
- and if so, track the response to that
- based on user input, communicate action taken in GUI - for clarity

In GUI this could be reported for example in brackets, what action was taken regarding that particular import. With that one could easily come back to that later, in case they wanted to take action again against that albu that was problematic, because currently such import will be displayed as "Completed".

Here's small demo on how could it look (excuse me but I didn't bother to change colors nor icon. Let's assume Abort could be in red, Skip in yellow/red, Use-as maybe tifferent hue of green etc.)

![obraz](https://github.com/sentriz/betanin/assets/135069967/8b4f57ca-3c18-4910-8218-13f3a538fd20)

My assumption here is that it's possible to capture whenever this interactive menu is displayed, and that user always provides predictable input within possible options. Maybe more thought should be provided into what appens if user misstypes etc.

**Comment by sentriz** [comment]
i agree it would be nice to have extra info, but if input is required you should get a notification like that. recently the beets UI was updated and betanin didnt updated. see https://github.com/sentriz/betanin/issues/114 for more info.
